### PR TITLE
LON-1045

### PR DESCRIPTION
**Pull request by SandraPG85**
Add open/create multicmaera limitation

### CODE REVIEW

File: VAS.Tests/Multimedia/TestMultimediaToolkit.cs [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+
+		[SetUp]
+		public void SetUp ()
+		{
+			mockLimitationService = new Mock<ILicenseLimitationsService> ();
+			App.Current.LicenseLimitationsService = mockLimitationService.Object;
+		}
+
+		[Test]
+		public void GetMultiPlayer_MulyiCameraLimitationActive_ThrowException ()
```
Comment (vguzmanp) [inline VAS.Tests/Multimedia/TestMultimediaToolkit.cs]: typo `mulyi` -> `multi`

File: VAS.Tests/Multimedia/TestMultimediaToolkit.cs [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+
+			// Act
+			try {
+				sut.GetMultiPlayer ();
+			} catch (InvalidOperationException) {
+				exceptionThrow = true;
+			}
+
+			// Assert
+			Assert.IsTrue (exceptionThrow);
```
Comment (vguzmanp) [inline VAS.Tests/Multimedia/TestMultimediaToolkit.cs]: Isn't it better to `Assert.Throws`?

File: VAS.Tests/Multimedia/TestMultimediaToolkit.cs [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+			// Assert
+			Assert.IsFalse (exceptionThrow);
+		}
+	}
+
+	class DummyMultiPlayer : IMultiVideoPlayer
+	{
+		public ObservableCollection<CameraConfig> CamerasConfig {
+			set {
+				throw new NotImplementedException ();
```
Comment (vguzmanp) [inline VAS.Tests/Multimedia/TestMultimediaToolkit.cs]: Lol, this doesn't implement anything. Wouldn't it be better to just mock it?

File: VAS.Tests/Services/TestVideoPlayerController.cs [hunk 4]
```diff
@@ -1996,10 +1996,12 @@ public void LoadPlaylistVideo_PlayerVMFileSetUpdated ()
 			Assert.AreSame (mfsNew [0], fileset.Model [0]);
 		}
 
+		[Test]
```
Comment (vguzmanp) [inline VAS.Tests/Services/TestVideoPlayerController.cs]: ?

File: VAS.UI.Gtk2/UI/Component/MediaFileChooser.cs [hunk 5]
```diff
@@ -52,7 +55,13 @@ public MediaFileChooser (String name)
 
 			UpdateFile ();
 
-			entrybutton_addbutton.Clicked += HandleAddClicked;
+			// FIXME: Use a viewmodel and the proper bindings
+			// Warning: Note also that if the index is incremented from another part than the new panel the limitation will be also
+			// applied there.
+			addLimitationCommand = new LimitationCommand (VASFeature.CreateMultiCamera.ToString (), () => HandleAddClicked (null, null));
+			addLimitationCommand.LimitationCondition = () => index > 1;
```
Comment (vguzmanp) [inline VAS.UI.Gtk2/UI/Component/MediaFileChooser.cs]: Why do you start in 1 for the index?

File: VAS.Tests/Multimedia/TestMultimediaToolkit.cs (hunk 3 above)
Comment (SandraPG85) [inline VAS.Tests/Multimedia/TestMultimediaToolkit.cs]: I need the type itself

File: VAS.Tests/Services/TestVideoPlayerController.cs (hunk 4 above)
Comment (SandraPG85) [inline VAS.Tests/Services/TestVideoPlayerController.cs]: changed by mistake, rollback

File: VAS.UI.Gtk2/UI/Component/MediaFileChooser.cs (hunk 5 above)
Comment (SandraPG85) [inline VAS.UI.Gtk2/UI/Component/MediaFileChooser.cs]: 1 means 1 camera or file

File: VAS.Tests/Multimedia/TestMultimediaToolkit.cs (hunk 3 above)
Comment (vguzmanp) [inline VAS.Tests/Multimedia/TestMultimediaToolkit.cs]: 👌

**Pull request merged**